org.mwc.cmap.legacy/src/MWC/GUI/BaseLayer.java

```diff
@@ -202,6 +202,11 @@
 	 */
 	private final boolean _orderedChildren;
 
+  /**
+   * property type to signify data being added or removed
+   */

```

Comment:
Hmm, @saulhidalgoaular we don't refer to this string - doesn't that suggest that no-one is listening to it?